README.md

```diff
@@ -75,6 +75,21 @@ improvement campaign_ analysis.
 > fixes to the default branch immediately, add a comment with the contents
 > "@pixeebot next" to any PR or issue in the repository.
 
+## For Other Tools
+
+For other supported tools that do not automatically fetch results, the `file` input can be used. In this case, it is assumed that the results file is checked into the repository and the provided path is relative to the repository root.
+
+```yaml
+- uses: pixee/upload-tool-results-action@v2
+  with:
+    # Use any supported SARIF tool here
```

Comment:
I think we should explicitly list the others (Snyk, CodeQL) or put a `|...` to imply it's good to put other values here.